apps/fulfillment/mock/routes.json

```diff
@@ -0,0 +1,4 @@
+{
+  "/picking-orders?filter[order.ids]=:id": "/orders/:id",
+  "/picking-orders/:id": "/orders/:id"
```

Comment:
There are only used in picking app and are not used in fulfillment app so please remove them.